stack/stack.c

```diff
@@ -0,0 +1,120 @@
+#include "../stack/stack.h"
```

Comment:
```suggestion
#include "stack.h"
```
"Выйди и зайди обратно" получается =)